proc/proc.go

```diff
@@ -151,6 +149,9 @@ func (dbp *Process) LoadInformation(path string) error {
 }
 
 func (dbp *Process) FindFileLocation(fileName string, lineno int) (uint64, error) {
+	if runtime.GOOS == "windows" {
```

Comment:
Unfortunately no - this is doing the opposite of that library function.  We need to turn a platform-specific path into one which always uses '/', since for some reason that is how paths are being encoded into DWARF, even on Windows.  

We could avoid the runtime check by always doing something like this - which will just be a no-op on unix:

```go
fileName = strings.Replace(fileName, string(os.PathSeparator), "/", -1)
```